compiler/rustc_target/src/spec/powerpc64le_unknown_linux_musl.rs

```diff
@@ -9,7 +9,7 @@ pub fn target() -> Target {
     Target {
         llvm_target: "powerpc64le-unknown-linux-musl".to_string(),
         pointer_width: 64,
-        data_layout: "e-m:e-i64:64-n32:64".to_string(),
+        data_layout: "e-m:e-i64:64-n32:64-v256:256:256-v512:512:512".to_string(),
```

Comment:
I have a suspicion that this won't work right with system LLVM that are <12.0, as IIRC we check that the `data_layout` is the same as what LLVM reports as the default. Not sure how to fix this though, if true.

Reply:
Nevermind, seems like its handled elsewhere!